upload_form.py

```diff
@@ -459,8 +550,11 @@ def set_columns(filename, fileformat=None):
                                                     branch=branch_database,
                                                    )
     except Exception as ex:
-        return render_template('error.html', error=str(ex),
-                               traceback=traceback.format_exc(ex))
+        if testmode:
+            raise ex
+        else:
+            return render_template('error.html', error=str(ex),
```

Comment:
Oh... right.  I hadn't appreciate that this change was returning `render_template` from a different function.  I'll come up with a solution.